src/functionalTest/resources/data/opa-press-list/opaPressList.json

```diff
@@ -37,6 +37,39 @@
                         "case": [
```

Comment:
There are 2 more OPA files in the functional test folders need updating

Reply:
Updated the 2 OPA files. Thanks Kian